src/System.Collections.NonGeneric/tests/Hashtable/ContainsValueTests.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.Text;
+using System.Collections;
+
+namespace System.Collections.HashtableTests
+{
+    public class ContainsValueTests
+    {
+        [Fact]
+        public void TestContainsValueBasic()
+        {
+            StringBuilder sblMsg = new StringBuilder(99);
+
+            Hashtable ht1 = null;
+
+            string s1 = null;
+            string s2 = null;
+
+            ht1 = new Hashtable(); //defalult constructor
```

Comment:
There are a couple of instances of this typo ("defalult" instead of "default"), it'd look tidier if they were cleaned up.